pages.it/linux/fsck.md

```diff
@@ -0,0 +1,15 @@
+# fsck
+
+> Controlla l'integrità di un filesystem o lo ripara. Il filesystem non dev'essere montato al momento in cui il comando viene eseguito.
+
+- Controlla il filesystem /dev/sda, riportando eventuali blocchi danneggiati:
+
+`fsck {{/dev/sda}}`
```

Comment:
We start using `sdX` as a placeholder for devices to avoid run a disk using a copy-paste to the terminal
```suggestion
`fsck {{/dev/sdX}}`
```
You can send the fix to the English page if you want in another PR.